dioptas/widgets/plot_widgets/HistogramLUTItem.py

```diff
@@ -89,6 +90,8 @@ def __init__(self, image=None, fillHistogram=False, orientation='horizontal', au
         self.gradient = GradientEditorItem()
         self.gradient.loadPreset('grey')
 
+        self._normalizationLabel = pg.LabelItem("linear")
```

Comment:
Is linear here correct? should the default not be "lin"? Also maybe lin could just be abreviated with "/"

Reply:
Indeed! That could even be `""` because it's not really meaningful without an image item.